libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -724,6 +734,7 @@ dictionary RecommendedFees {
 [Enum]
 interface InputType {
     BitcoinAddress(BitcoinAddressData address);
+    LiquidAddress(LiquidAddressData address);
```

Comment:
Still propagating to the sdk here. since we build the greenlight sdk I think we can remove it from here.